project_task_add_very_high/models/project_task.py

```diff
@@ -7,4 +7,6 @@
 class ProjectTask(models.Model):
     _inherit = "project.task"
 
-    priority = fields.Selection(selection_add=[("2", "High"), ("3", "Very High")])
+    priority = fields.Selection(
+        selection_add=[("2", "High"), ("3", "Very High")], tracking=True
```

Comment:
This new functionality should be mentioned in the readme

Reply:
Thanks, I'll add it now)